shell.lint.cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+- id: 'Lint Shell'
+  name: 'koalaman/shellcheck-alpine'
+  entrypoint: '/bin/sh'
+  args:
+  - -c
+  - |
+    for f in $(find . -name '*.sh' -not -path "./testing/*")
+      do shellcheck -f gcc $f
+    done
```

Comment:
I don't have strong opinions on this, but here's the output of running shellcheck on these lines:

```
In s.sh line 2:
for f in $(find . -name '*.sh' -not -path "./testing/*")
         ^-- SC2044: For loops over find output are fragile. Use find -exec or a while read loop.


In s.sh line 3:
     do shellcheck -f gcc $f
                          ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
     do shellcheck -f gcc "$f"

For more information:
  https://www.shellcheck.net/wiki/SC2044 -- For loops over find output are fr...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

For the first one, I don't have a strong opinion but if we're going to showcase this maybe we should see if we can reduce this rule to a warning.

For the second, we actively want globbing so not sure what to make of this complaint.

I think this is along the lines of what it prefers by default:
```
find . -name '*.sh' -not -path "./testing/*" -exec shellcheck -f gcc {} \;
```

Reply:
Nice, thanks. Also, globbing is automatically set as a warning.